misc/dist/macos_tools.app/Contents/Info.plist

```diff
@@ -19,7 +19,7 @@
 	<key>CFBundleShortVersionString</key>
 	<string>4.3</string>
 	<key>CFBundleSignature</key>
-	<string>godot</string>
+	<string>GODO</string>
```

Comment:
This generates a warning, as `CFBundleSignature`s can only be 4 characters

Reply:
I'd suggest splitting the changes to that file to a separate PR, as it seems like things we'd need regardless of Metal.